lib/kennel.rb

```diff
@@ -80,12 +80,26 @@ def update
 
     private
 
+    def download_definitions
+      Progress.progress("Downloading definitions", plain: true) do
```

Comment:
The PR description shows this line appearing first and list, sandwiching the `generate` stuff. However that's not guaranteed; there's a race condition. Combine that with the fact that this progress is "plain" but not all of the inner ones are, and the output might end up looking odd.

Maybe that's fine though.

Reply:
- there should be no inner progress since there are only these 2 in parallel
- they will appear in a random order, not really a big issue afaik